src/context.h

```diff
@@ -131,6 +131,8 @@ typedef struct getdns_upstream {
 	/* How is this upstream doing over UDP? */
 	int                      to_retry;
 	int                      back_off;
+	size_t                   udp_responses;
+	size_t                   udp_timeouts;
```

Comment:
I was actually wondering if it would be useful to add these (and the TCP/TLC connection info) to the call_reporting dict? If you don't think so I will wrap them for now.